### PR TITLE
[SIG-3268] Adds a new `alert-inverted` style to the PlainText component

### PR DESCRIPTION
**Pull request by aditudorache**
This PR adds a new `alert-inverted` style to the PlainText component.

### CODE REVIEW

File: src/signals/incident/definitions/wizard-step-2-vulaan/wegen-verkeer-straatmeubilair.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ export const controls = {
   extra_kerstverlichting: {
     meta: {
       label: '',
```
Comment (MauritsR) [inline src/signals/incident/definitions/wizard-step-2-vulaan/wegen-verkeer-straatmeubilair.js:27]: Although not part of this PR, I missed these things in the previous PR for this story: 

- Label key can be left out altogether
- `['lantaarnpaal-straatverlichting']` can be `'lantaarnpaal-straatverlichting'`

**Review by MauritsR — APPROVED** [review]
Looks good

File: src/signals/incident/definitions/wizard-step-2-vulaan/wegen-verkeer-straatmeubilair.js (hunk 1 above)
Comment (aditudorache) [reply src/signals/incident/definitions/wizard-step-2-vulaan/wegen-verkeer-straatmeubilair.js]: Good point, I changed this as wel!
